package.json

```diff
@@ -20,15 +20,13 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "broccoli-funnel": "^1.2.0",
-    "broccoli-merge-trees": "^2.0.0",
     "ember-cli-babel": "^6.3.0",
     "jwt-decode": "^2.2.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
     "ember-ajax": "^3.0.0",
-    "ember-cli": "~2.14.1",
+    "ember-cli": "~3.0.0",
```

Comment:
Can we not upgrade ember in this pr? As we are missing some upgrades diff

Reply:
Or better let's add those missing different :)

Reply:
Sorry, I'm not sure I'd have the time to help with that!

Upgrading to Ember CLI 2.15 is necessary for this fix to work, so I've made changes to only upgrade to 2.15. No other big changes happened between 2.14 and 2.15, so it should be okay. https://www.emberjs.com/blog/2017/09/01/ember-2-15-released.html

Reply:
Let me finish it for as it still not mergeable. The upgrade is not fully accomplished. https://github.com/ember-cli/ember-addon-output/compare/v2.14.0...v2.15.0